pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -175,6 +176,33 @@ Avo will override all events in the RudderStack Tracking Plan that are being imp
 
 Avo will not override any events that are only defined in RudderStack, or events that have not been excluded from the RudderStack integration in Avo with filters.
 
+<img
+  src={require('../../../images/publishing_mparticle.svg')}
+  alt="mParticle"
+  style={{ margin: '32px 0 -32px 0' }}
+/>
+
+## <a name="mparticle"></a> mParticle Data Master
+
+The [mParticle integration](https://docs.mparticle.com/guides/data-master/introduction/) enables you to publish your Avo Tracking Plan into mParticle. You can configure the integration to publish whenever a branch is merged or with a push of a button.
```

Comment:
I assumed the link would take me to some detailed material about the Avo<>mParticle integration, I suggest this change to better clarify where the links take you:
```suggestion
The [mParticle Data Master](https://docs.mparticle.com/guides/data-master/introduction/) integration enables you to publish your Avo Tracking Plan into mParticle Data Master. You can configure the integration to publish whenever a branch is merged or with a push of a button.
```